packages/mobile/src/components/CodeInput.tsx

```diff
@@ -109,18 +136,24 @@ export default function CodeInput({
                 // numberOfLines is currently Android only on TextInput
                 // workaround is to set the minHeight on iOS :/
                 numberOfLines={Platform.OS === 'ios' ? undefined : numberOfLines}
-                inputStyle={
-                  Platform.OS === 'ios' && numberOfLines
-                    ? {
-                        minHeight: LINE_HEIGHT * numberOfLines,
-                      }
-                    : undefined
-                }
+                inputStyle={{
+                  ...(shortVerificationCodesEnabled && {
+                    ...fontStyles.large,
+                    textAlign: 'center',
+                  }),
+                  minHeight:
+                    Platform.OS === 'ios' && numberOfLines
+                      ? LINE_HEIGHT * numberOfLines
+                      : undefined,
+                }}
                 autoCapitalize="none"
                 testID={testID}
               />
```

Comment:
I think we also want `keyboardType` to be `number-pad` when `shortVerificationCodesEnabled` is `true`, right?